src/x-platform/electron.js

```diff
@@ -17,6 +18,24 @@ if (window.require) {
 
 const notificationClickTimeout = 20000
 
+// http://crocodillon.com/blog/parsing-emoji-unicode-in-javascript
+const emojiRegExp = new RegExp([
```

Comment:
it is not 3, it is range of unicode symbols, that converts all emoji's, details by the link above